src/test/java/edu/harvard/iq/dataverse/api/MetricsIT.java

```diff
@@ -30,7 +30,7 @@ public static void cleanUpClass() {
 
     @Test
     public void testGetDataversesToMonth() {
-        String yyyymm = "2018-04";
+        String yyyymm = "2023-11";
```

Comment:
if it's important for these dates to be current, should we have it instead get the values from Date.now()?

Reply:
Yeah, I was just being a little lazy - trying to get somewhat meaningful results without changing too much. I fixed it.